lldb/source/Plugins/LanguageRuntime/CPlusPlus/ItaniumABI/ItaniumABILanguageRuntime.cpp

```diff
@@ -191,14 +191,6 @@ llvm::Error ItaniumABILanguageRuntime::TypeHasVTable(CompilerType type) {
       type = pointee_type;
   }
 
-  // Make sure this is a class or a struct first by checking the type class
-  // bitfield that gets returned.
-  if ((type.GetTypeClass() & (eTypeClassStruct | eTypeClassClass)) == 0) {
```

Comment:
I might have overgeneralized here a bit - `TypeSystemClang::IsPolymorphicClass` already checks these conditions, and seems to handle situations better where `CompilerType` is a typedef to a struct/class instead of being one itself.